platform/golden/provisioners/01_update_packer_user/roles/packer-finalize/tasks/main.yml

```diff
@@ -31,9 +31,23 @@
     upgrade: true
 
 # Fix grub config to support tty at boot
+- name: add iscsi packages
+  ansible.builtin.apt:
+    name: open-iscsi
+    state: present
 
 - name: Update grub2 config to support tty at boot
   ansible.builtin.lineinfile:
     path: /etc/default/grub
-    line: 'GRUB_CMDLINE_LINUX="$GRUB_CMDLINE_LINUX console=ttyS0"'
-  notify: update grub
+    line: 'GRUB_CMDLINE_LINUX=$GRUB_CMDLINE_LINUX crashkernel=auto ip=dhcp LANG=en_US.UTF-8 console=tty0 console=ttyS0 iommu=on netroot=iscsi:@169.254.0.2::::iqn.2015-02.oracle.boot:uefi iscsi_initiator=iqn.2015-02.oracle.boot:instance biosdevname=0'
+
+- name: Re-install grub2 efi
```

Comment:
what went wrong the first efi installation? A system hsould never need a reinstall of grub after inital intallation.

It looks liek like 42, you wish to override `netroot` and turn the `iommu=on`. Unfortunately thhis `IOMMU` is one of the things I was talking about that was a blocker.

What I suggest is booting a ubuntu image in OCI. You will find a file in `/etc/default/grub.d` that adds to the grub command line.

The features it adds are iscsi related, although this is from memory.

Reply:
The SATA line in the iso controller was “required” to support EFI boot for the OS (the bootloader is in EFI mode, and the image has EFI bios and disk partitions).  This was called out in the `virtualbox-iso` packer documentation as a requirement to get the system to boot via EFI _after_ rebooting without the CD attached.